src/common/SVGWidget.js

```diff
@@ -120,6 +120,12 @@
         return this;
     };
 
+    SVGWidget.prototype.rotation = function (_) {
+        if (!arguments.length) return this._rotation;
+        this._rotation = _;
+        return this;
+    };
+    
```

Comment:
If rotation is not applied within Widget then the setter/getter should not be declared here (IOW it should be in Text.js)